request/ta_send_mam.h

```diff
@@ -27,7 +27,7 @@ extern "C" {
 typedef struct send_mam_req_s {
   tryte_t prng[NUM_TRYTES_ADDRESS + 1];
   char* payload;
-  uint8_t channel_ord;
+  int32_t channel_ord;
```

Comment:
Why did you promote from `uint8_t` to `int32_t`? Please be aware of the signedness.

Reply:
`channel_ord` is in `trint18_t` in entangled. According to MAM spec `trint18_t` is a type in the range from +- (3^18-1)/2, so that is why I use `int32_t` intead of `uint8_t`

Reply:
Mention it within git commit messages.